packages/ts-migrate-plugins/src/plugins/add-conversions.ts

```diff
@@ -10,11 +10,11 @@ type Options = {
 const addConversionsPlugin: Plugin<Options> = {
   name: 'add-conversions',
   run({ fileName, sourceFile, text, options, getLanguageService }) {
-    // Filter out TS2339: Property '{0}' does not exist on type '{1}'.
+    // Filter out diagnostics we care about.
     const diags = getLanguageService()
       .getSemanticDiagnostics(fileName)
       .filter(isDiagnosticWithLinePosition)
-      .filter((diag) => diag.code === 2339);
+      .filter((diag) => diag.code === 2339 || diag.code === 2571);
```

Comment:
nit: maybe we can move diagnostics values to the separate structure, like set or array, for the future extendability:

`const supported = new Set([2339, 2571])`;
`.filter((diag) => supported.has(diag.code));`